fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -280,12 +280,14 @@ public List<UncertainPairedData> Compute(IProvideRandomNumbers randomProvider)
             List<UncertainPairedData> results = ConsequenceDistributionResults.ToUncertainPairedData(allStagesAtIndexLocation, consequenceDistributionResults);
             return results;
         }
-        public float[] ExtrapolateFromAboveAtIndexLocation(float[] stagesAtStructuresHighestProfile, float upperInterval, int stepCount)
+        public static float[] ExtrapolateFromAboveAtIndexLocation(float[] stagesAtStructuresHighestProfile, float upperInterval, int stepCount)
```

Comment:
static for testing

Reply:
If you're making something public or static for testing, make sure you leave a comment on it. We really need to be careful doing that. It makes our code confusing.  There is a way to test private methods. I've never done it, but I'll share some info in rocketchat or github.